content/en/_index.md

```diff
@@ -163,6 +180,42 @@ public static void listInfoTypes() throws IOException {
   }
 }
 {{< /tab >}}
+{{< tab header="Python" >}}
+def list_info_type(
+    language_code: Optional[str] = "en-US",
+    result_filter: Optional[str] = "supported_by=INSPECT",
+):
+    """List types of sensitive information within a category.
+
+    Args:
+        language_code (str): The BCP-47 language code to use, e.g. 'en-US'.
+        filter (str): An optional filter to only return info types supported
+            by certain parts of the API. Supported filters are 
+            "supported_by=INSPECT" and "supported_by=RISK_ANALYSIS". Defaults
+            to "supported_by=INSPECT".
+    """
+    # Initialize client that will be used to send requests. This client only
+    # needs to be created once, and can be reused for multiple requests. After
+    # completing all of your requests, call the "__exit__" method on the client to
+    # safely clean up any remaining background resources. Alternatively, use the
+    # client as a context amnager.
+    dlp_client = dlp_v2.DlpServiceClient()
```

Comment:
1. These are specifically not threadsafe, right?
2. Does the user need to call __exit__? or is it called on garbage collection?

Reply:
1. Sharing across threads is okay, for multiprocessing we advise creating separate clients. I've re-worded the comment blurb to have a shorter version of this:

```
   Because this client uses :mod:`grpc` library, it is safe to
   share instances across threads. In multiprocessing scenarios, the best
   practice is to create client instances *after* the invocation of
   :func:`os.fork` by :class:`multiprocessing.pool.Pool` or
   :class:`multiprocessing.Process`.
```

3. Technically no, the gRPC channels will get cleaned up automatically Manually closing is useful if the user is in a situation where they are creating many clients or have a long running process and want the option to close off clients.

It looks like gRPC has resolved the issues that were part of the reason for introducing an explicit `close`, but I think they prefer we recommend this to folks.

* https://github.com/grpc/grpc/issues/22851
* https://github.com/grpc/grpc/issues/22123
* https://github.com/grpc/grpc/pull/22855

Reply:
Is sharing threads okay for just GAPIC clients or are the HTTP ones threadsafe as well now? I know for a while they weren't

Reply:
I honestly am not sure about the HTTP ones, it might differ depending on the client.

Reply:
So according to this: https://googleapis.github.io/google-api-python-client/docs/thread_safety.html#the-httplib2http-objects-are-not-thread-safe

It looks like they aren't'. Should we have different preferred text for both?